From 3d48de758554ea92a7a17ba5593602f943f9c785 Mon Sep 17 00:00:00 2001
From: Levi Armstrong <levi.armstrong@swri.org>
Date: Sun, 10 Jan 2021 19:27:01 -0600
Subject: [PATCH] Added install rule for package.xml

---
 CMakeLists.txt | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index adb111f..1976f3e 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -124,6 +124,9 @@ INSTALL(DIRECTORY include/qpOASES
   FILES_MATCHING PATTERN "*.hpp"
   PATTERN "*.ipp"
   PATTERN ".svn" EXCLUDE)
+  
+# Install catkin package.xml
+install(FILES package.xml DESTINATION share/qpoases)
 
 ############################################################
 ######################### examples #########################
-- 
2.25.1

